### DISCUSSION TITLE
Scores for the retrieved passages

### DISCUSSION THREAD

**Discussion by c3-pakazad** [stanford-futuredata/ColBERT#220] [Q&A]
I have a question concerning the approach for computing the scores for the retrieved docs. I have a text query with _N_ tokens, where _N_ <= query_maxlen. Assuming that attend_to_mask_tokens=False, do we expect the scores associated with documents retrieved for this query to be less than _N_ or less than query_maxlen?

**Comment by okhat** [comment]
attend_to_mask_tokens=False doesn't affect this.

So, "less than query_maxlen"

**Comment by okhat** [comment]
attend_to_mask_tokens=False is about earlier layers in the transformer, not the MaxSim scoring of colbert
